internal/server/data/query.go

```diff
@@ -103,12 +122,21 @@ func update(tx WriteTxn, item Updatable) error {
 	query.B(item.Table())
 	query.B("SET")
 	query.B(columnsForUpdate(item), item.Values()...)
-	query.B("WHERE deleted_at is null AND id = ?;", item.Primary())
+	query.B("WHERE deleted_at is null")
```

Comment:
Could hiding deleted at on update cause issues with restoring deleted rows?

Reply:
We don't have a way to restore deleted rows through the API, right? If we wanted to do that we'd have to write a migration or do it directly in the database?

My rational for `deleted_at is null` is that we don't really want someone to be able to updated the `deleted_at` field of a row that was already deleted.

If we ever support un-delete from the API we will definitely need to change this.